tests/test_fyle/conftest.py

```
@@ -65,3 +66,100 @@ def add_webhook_attribute_data(db):
+    expense_group = ExpenseGroup.objects.create(
+        workspace_id=workspace.id,
+        fund_source='PERSONAL',
+        description={'employee_email': expense.employee_email},
+        employee_name=expense.employee_name
+    )
+    expense_group.expenses.add(expense)
+    return expense_group
+
+
+@pytest.fixture
+def add_category_mapping_error(db, add_category_test_expense_group):
```

Ruff (0.14.8)

Unused function argument: `db`

(ARG001)